packages/core/test/lib/hint.test.ts

```diff
@@ -1,6 +1,6 @@
-import { captureEvent, getCurrentScope } from '@sentry/core';
 import { GLOBAL_OBJ } from '@sentry/utils';
 
+import { captureEvent, getCurrentScope } from '../../src';
```

Comment:
I actually fixed a bunch of tests along there way here we we used to import from the package itself (we are in core here), which lead to tests only running on the built output, which is a bit weird IMHO.